README.md

```diff
@@ -84,11 +84,8 @@ If this is not possible, add `$TENSORFLOW_SRC/bazel-bin/tensorflow` to
 
 You may need to run `ldconfig` to reset `ld`'s cache after copying `libtensorflow.so`.
 
-**OSX Note**: If you are running on OSX, there is a
-[Homebrew PR](https://github.com/Homebrew/homebrew-core/pull/10273) in process which, once merged,
-will make it easy to install `libtensorflow` wihout hassle. In the meantime, you can take a look at
-[snipsco/tensorflow-build](https://github.com/snipsco/tensorflow-build) which provides a homebrew
-tap that does essentially the same.
+**macOS Note**: Via [Homebrew](https://brew.sh/) in process which, you can just run
+`brew install libtensorflow`.
 
```

Comment:
How would you rewrite that section? I am unsure what to particularly add or remove.